node/sc/sub_bridge_handler.go

```diff
@@ -167,11 +174,33 @@ func (sbh *SubBridgeHandler) getRemoteGasPrice() uint64 {
 	return sbh.remoteGasPrice
 }
 
+// setRemoteGasPrice sets parent chain's gasprice
 func (sbh *SubBridgeHandler) setRemoteGasPrice(gasPrice uint64) {
 	sbh.subbridge.bridgeAccounts.pAccount.SetGasPrice(big.NewInt(int64(gasPrice)))
 	sbh.remoteGasPrice = gasPrice
 }
 
+// setRemoteChainValues sets parent chain's configuration values
+func (sbh *SubBridgeHandler) setRemoteChainValues(pcInfo parentChainInfo) {
+	headerNum := sbh.subbridge.blockchain.CurrentHeader().Number
+	sbh.setRemoteGasPrice(pcInfo.GasPrice)
+	if sbh.subbridge.blockchain.Config().IsKIP71ForkEnabled(headerNum) {
+		// Set parent chain's gasprice with upperboundbasefee
+		sbh.remoteGasPrice = pcInfo.KIP71Config.UpperBoundBaseFee
```

Comment:
```suggestion
		sbh.setRemoteGasPrice(pcInfo.KIP71Config.UpperBoundBaseFee)
```

Reply:
Ah.. okay..!! i don't have clear concept of how service chain works.. haha ;-;
If that was your intention, then fine :) Thank you for explanation!